db/migrate/20191118142930_create_social_profiles.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSocialProfiles < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users_social_profiles do |t|
+      t.references :user, foreign_key: true
+      t.integer :provider
+      t.string :nickname
+      t.string :uid
```

Comment:
Need `profile_url` and `image_url` fields here.  In fact let's also include `email` here too, and delegate to it from `User`.